frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -48,45 +47,92 @@ const Navbar = ({children}) => {
                 </p>
               ))}
-              {isLoggedIn ? 
+              {isLoggedIn ? (
                 <div className="relative">
-                  <div onClick={()=>{setUserMenuOpen(prev => !prev)}} className="cursor-pointer">
+                  <button
+                    onClick={() => {
+                      setUserMenuOpen((prev) => !prev);
+                    }}
+                    className="cursor-pointer"
+                  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)